### PR TITLE
fix(package): update to google-auth-library 1.0

### PR DESCRIPTION
**Pull request by ofrobots**
~~**DO NOT LAND YET**. Opening the PR early to exercise the tests, and as a heads up. Please see the [release notes](https://github.com/google/google-auth-library-nodejs/releases/tag/1.0.0-alpha.1).~~

google-auth-library 1.0 has some breaking changes. This PR updates
the uses of google-auth-library in this module.

### CODE REVIEW

**Comment by ofrobots** [comment]
This is ready for review. PTAL.

File: utils/auth.js [hunk 1]
```diff
@@ -16,10 +16,9 @@
 
 'use strict';
 
-const GoogleAuth = require('google-auth-library');
+const {OAuth2Client} = require('google-auth-library');
```
Comment (Canain) [inline utils/auth.js]: Hi, we just updated the linter to enforce spacing for destructuring objects. It was to enforce consistency for module imports like this as when there are multiple destructured properties, it is more readable with open spacing.

Can you change it to:
```javascript
const { OAuth2Client } = require('google-auth-library');
```

Comment (ofrobots) [reply utils/auth.js]: Done.

**Comment by smishra2** [comment]
According to the Google Auth Library [code](https://github.com/google/google-auth-library-nodejs/blob/7e9a3903c14340bc20ec4935138e6fb0dd8d1f8a/src/auth/oauth2client.ts#L723), we might be using an outdated form of this method, but it should work til version 2.x

**Comment by ofrobots** [comment]
Can this PR be landed now?

**Comment by smishra2** [comment]
This unfortunately did turn out to be a breaking change, per Issue #92. Working on a patch fix.

**Comment by smishra2** [comment]
Patch has been submitted, diff is [here](https://github.com/actions-on-google/actions-on-google-nodejs/commit/21792359aafba1609e3a0055a67d676311166d79)

**Pull request merged**